README.md

```diff
@@ -1,66 +1,144 @@
+All the generated files are stored in the `tmp` folder under the project root:
 
-## Running the app
+```
+tmp
+├── json-docs   // JSON:API-comlaint docs generated locally from YUIDoc files in `s3-docs`.
+├── rev-index   // Rev index files used for generating JSON:API-comlaint docs.
+├── s3-docs     // YUIDoc docs generated locally or downloaded from s3://api-docs.emberjs.com.
+```
 
-1. Fork/Clone [ember-jsonapi-docs](https://github.com/ember-learn/ember-jsonapi-docs)
-1. Run `yarn`
-1. Install the [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
-1. Set up AWS access
+> ℹ️ **NOTE:** If you are looking for the app behind https://api.emberjs.com/, visit [ember-api-docs](https://github.com/ember-learn/ember-api-docs) instead.
 
-   ```shell
-   export AWS_ACCESS_KEY_ID=xxxxxx
-   export AWS_SECRET_ACCESS_KEY=xxxxx
-   ```
+## Prerequisites
 
-   The app accesses builds.emberjs.com (an Amazon S3 bucket) in read-only mode, which is public. This requires any valid AWS credentials.
+1. Install the latest [Node.js](https://nodejs.org/) LTS.
+2. Install [AWS CLI version 2](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
+3. Create an account in [AWS Management Console](https://console.aws.amazon.com). [Create a New Access Key](https://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html) (access key ID and secret access key) in *My Security Credentials* under your profile name to be able to access public AWS S3 buckets.
 
-   You can get your credentials by logging into your [AWS console](https://console.aws.amazon.com) and navigating to "_My Security Credentials_" under your profile name. You can generate a new pair under the "_Access Keys (Access Key ID and Secret Access Key)_" section.
+## Quickstart
 
-1. To test your changes in the app run,
-   `yarn start`
-   The app tries to process all ember & ember-data versions since 1.0 which takes high memory & time to complete. If you intend it, then run `yarn start --max_old_space_size=8192`.
-   You are setting your node max heap space to 8GB, so make sure you have that much space available on your machine.
+### Generate Docs from Ember Source
 
-## To Generate docs for a specific project and/or version for development
+You can generate docs from your local copies of [emberjs/ember.js](https://github.com/emberjs/ember.js/) and [emberjs/data](https://github.com/emberjs/data) repositories and serve them locally as [JSON:API](http://jsonapi.org/).
 
-You can do this by passing `--project ember/ember-data --version 2.11.1` as an argument to the index script. e.g., `yarn start --project ember --version 2.11.0`.
-You need an additional flag `AWS_SHOULD_PUBLISH=true` for publishing the docs.
+```bash
+# Clone Ember.js and Ember Data repositories.
```

Comment:
We should make the structure of the docs clear. ember.js, data, ember-api-docs, and ember-jsonapi-docs should all be cloned in the same directory.

Reply:
That's an excellent point! I'll create another PR to include this information.